src/types/global.ts

```diff
@@ -441,6 +454,7 @@ declare global {
 			notificationEventHistory?: HeaderNotification[][];
 			commercialTimer?: EventTimer;
 			offlineCount?: number;
+			identityAuth?: IdentityAuth<AccessTokenClaims, CustomIdTokenClaims>;
```

Comment:
```suggestion
			identityAuth?: IdentityAuth<never, CustomIdTokenClaims>;
```

You only need to pass on any additional custom claims to extend the type, the default access token claims are always provided